vertx-db2-client/src/test/java/io/vertx/db2client/DB2ErrorMessageTest.java

```diff
@@ -26,66 +26,155 @@
 
 @RunWith(VertxUnitRunner.class)
 public class DB2ErrorMessageTest extends DB2TestBase {
-	
+
 	@Test
-	@Ignore		// TODO - Need to figure out why this is blowing up in parseACCSECreply()
 	public void testConnectInvalidDatabase(TestContext ctx) {
-	  options.setDatabase("DOES_NOT_EXIST");
-	  DB2Connection.connect(vertx, options, ctx.asyncAssertFailure(err -> {
-		  ctx.assertTrue(err instanceof DB2Exception, "The error message returned is of the wrong type.  It should be a DB2Exception, but it was of type " + err.getClass().getSimpleName());
-//		  DB2Exception ex = (DB2Exception) err;
-//		  ctx.assertTrue(ex.getMessage().contains("Invalid database"), "The SQL error message returned is not correct.  It should have contained \"Invalid database\", but instead it said \"" + ex.getMessage() + "\"");
-	  }));
+		options.setDatabase("DB_DOES_NOT_EXIST");
+		DB2Connection.connect(vertx, options, ctx.asyncAssertFailure(err -> {
+			ctx.assertTrue(err instanceof DB2Exception, "The error message returned is of the wrong type.  It should be a DB2Exception, but it was of type " + err.getClass().getSimpleName());
+			DB2Exception ex = (DB2Exception) err;
+			ctx.assertTrue(ex.getMessage().contains("provided was not found") ||
+					ex.getMessage().contains("The connection was closed by the database server"), 
+					"The SQL error message returned is not correct.  It should have contained \"provided was not found\" or \"The connection was closed by the database server\", but instead it said \"" + ex.getMessage() + "\"");
+			ctx.assertTrue(ex.getErrorCode() == SqlCode.DATABASE_NOT_FOUND ||
+					ex.getErrorCode() == SqlCode.CONNECTION_REFUSED, 
+					"Wrong SQL code received");
+			ctx.assertTrue(ex.getSqlState().equalsIgnoreCase("2E000") ||
+					ex.getSqlState() == SQLState.AUTH_DATABASE_CONNECTION_REFUSED,
+					"Wrong SQL state received");
+//			System.out.println(ex.getMessage());
```

Comment:
all of these commented out print statements can be removed

Reply:
Done